pandas/tests/groupby/test_function.py

```diff
@@ -1574,3 +1574,11 @@ def test_corrwith_with_1_axis():
     )
     expected = Series([np.nan] * 6, index=index)
     tm.assert_series_equal(result, expected)
+
+
+def test_multiindex_group_all_columns_when_empty():
+    # GH 32464
+    df = DataFrame({"a": [1], "b": [2], "c": [3]}).set_index(["a", "b", "c"])
+    result = df.groupby(["a", "b", "c"]).sum().index[:0]
```

Comment:
This was my error - it should be `df[:0].groupby...` instead of at the end. We want result to be from evaluating groupby on an empty frame.

Reply:
You are absolutely right! In that case, I will further simplify the test by directly making the frame empty upon creation.

Let me know if that makes sense to you - thanks!

Reply:
Ah - yes, that would be simpler. Sounds great.